.github/workflows/check-api-data-in-sync.yml

```diff
@@ -0,0 +1,24 @@
+# This action fails if the API docs JSON data files have changed but have not
+# been committed.
+
+# We check generated API data into the Git repo because otherwise it is very
+# difficult to tell when something has changed or broken in them. It also makes
+# deploying faster, since we don't need to clone and analyze repos every time.
```

Comment:
Does this mean if you are doing an unrelated change and the API docs change, you need to include the API change or be blocked from merging by Github Actions?

Worth adding a comment here for how to unblock yourself if this is failing?

Reply:
The API docs shouldn't change if you're doing something unrelated. You'd have to change the API generating code, or the config that determines which repo + commit we are analyzing.